portal/src/main/webapp/error.html

```diff
@@ -0,0 +1,13 @@
+<html>
```

Comment:
Apparently the RequestRejectedException is a way of handling potentially dangerous urls. As a result, I felt that the url itself was potentially dangerous, so I didn't want to use it to reference any other paths. That's why I chose to not pull in any css / images or use an existing jsp.